website/content/en/preview/concepts/disruption.md

```diff
@@ -87,6 +87,8 @@ spec:
 
 ### Consolidation
 
+Consolidation is configured by `consolidationPolicy` and `consolidateAfter`. `consolidationPolicy` determines the pre-conditions for nodes to be considered consolidatable, and are `whenEmpty` or `whenEmptyOrUnderutilized`. If a node has no running non-daemon pods, it is considered empty.  `consolidateAfter` can be set to indicate how long Karpenter should wait after a pod schedules or is removed from the node before considering the node consolidatable. With `whenEmptyOrUnderutilized`, Karpenter will consider a node consolidatable when its `consolidateAfter` has been reached, empty or not.
```

Comment:
IMHO: would be nice to include an example (can re-use one of the existing examples and simply amend it) of how to specify consolidateAfter. It would make it easier to discover the right format for the value.